pkg/test/framework/runtime/components/apps/kube.go

```diff
@@ -302,26 +324,62 @@ func (c *kubeComponent) Start(ctx context.Instance, scope lifecycle.Scope) (err
 		return err
 	}
 	c.env = env
+	namespace := env.NamespaceForScope(scope)
+
+	// If test not require their apps explicitly, we deploy a suite of default apps.
```

Comment:
If the test does not explicitly describe the apps it needs, deploy a suite of default apps.

Reply:
Changed.